deploy/kubernetes/overlays/stable/ecr/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ bases:
 images:
   - name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
     newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver
+    newTag: v1.5.0
```

Comment:
also update the new gcr overlay tag (it was just merged yesterday )

Reply:
Done